### ISSUE TITLE
AR_data for custom synthetic data made with Isaac Sim Replicator

### ISSUE THREAD

**Issue by monajalal** [NVIDIA-ISAAC-ROS/isaac_ros_pose_estimation#34]
Hello,

How can I create AR_data for the json file for custom synthetic dataset so that I can use it for CenterPose evaluator?

Currently, CenterPose evaluator expects AR_data as a field in the json annotation.

**Comment by mujiwob** [comment]
@monajalal Hi, do you know how to get AR_data now? I also encountered this problem when training in TAO centerpose.

**Comment by Jason-xy** [comment]
@monajalal @mujiwob Hi, did you find any solution to get AR_data for the json file? I have encountered the same issue while training in TAO centerpose.

**Comment by mujiwob** [comment]
> @monajalal @mujiwob Hi, did you find any solution to get AR_data for the json file? I have encountered the same issue while training in TAO centerpose.

No. I found that AR_data is part of the Objectron dataset, but I don't know how to get it in the synthetic dataset by Isaac Sim. 
I just commented out the [relevant code](https://github.com/NVIDIA/tao_pytorch_backend/blob/main/nvidia_tao_pytorch/cv/centerpose/utils/centerpose_evaluator.py#L279) in centerpose_evaluator.py to skip this part, which made the training go on. However, the inference results after training are not good, I don't know if it is because of the lack of AR_data or my dataset is not good enough.

**Comment by Jason-xy** [comment]
I have bypassed this problem by assigning all fields in `AR_data` to `0`, and then trained on my dataset of 20k images. finally,  the results were good, so I guess `AR_data` has no impact on the current training process.

**Comment by mujiwob** [comment]
@Jason-xy So I think my poor results are probably due to the dataset. Is your dataset generated by Isaac Sim? If so, could you share the yaml setting files for generating the data and training? Thanks!

**Comment by monajalal** [comment]
> I have bypassed this problem by assigning all fields in `AR_data` to `0`, and then trained on my dataset of 20k images. finally, the results were good, so I guess `AR_data` has no impact on the current training process.

did you train using NVIDIA TAO toolkit? Where did you set it to 0?
Could you show you command for running the training?
